### PR TITLE
Create ita-j-agronomy.csl

### PR DESCRIPTION
**Pull request by vidotto**
Created a style file for the scientific journal ITALIAN JOURNAL OF AGRONOMY

### CODE REVIEW

File: ita-j-agronomy.csl [hunk 1]
```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-GB">
+  <info>
+    <title>Italian Journal of Agronomy</title>
+    <id>http://www.zotero.org/styles/ita-j-agronomy</id>
+    <link href="http://www.zotero.org/styles/ita-j-agronomy" rel="self"/>
```
Comment (damnation333) [inline ita-j-agronomy.csl]: can you write the ID, filename and self-link with the full name? italian-journal-of-agronomy

File: ita-j-agronomy.csl [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    <link href="http://www.zotero.org/styles/springer-basic-author-date" rel="template"/>
+    <author>
+      <name>Francesco Vidotto</name>
+      <email>francesco.vidotto@unito.it</email>
+    </author>
+    <contributor>
+      <name>Francesco Vidotto</name>
+      <uri>http://www.mendeley.com/profiles/francesco-vidotto/</uri>
+    </contributor>
```
Comment (damnation333) [inline ita-j-agronomy.csl]: duplication there. You can add the uri within your author bit.

File: ita-j-agronomy.csl [hunk 3]
```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-GB">
+  <info>
+    <title>Italian Journal of Agronomy</title>
+    <id>http://www.zotero.org/styles/ita-j-agronomy</id>
```
Comment (damnation333) [inline ita-j-agronomy.csl]: You can add a title-short here with IJA.

**Comment by damnation333** [comment]
You'll need to change the file name, too, to get it to validate. ;)

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**